src/assets/images/tally.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="300" height="300" viewBox="0 0 300 300" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
We don't want to add anything in the `src/assets` because this is a fork. We only add in `src/custom`

Reply:
reverted